### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by ab-noori**
### 🌟Branch features:

- JavaScript file divided into modules.
- For each class, separate modules are created.
- SPA navigation menu added.
- Header and footer added.
- All of the functions changed to arrow functions.
- Luxon library added and its object is used in the project.

### [ 🚀 Branch Demo](https://ab-noori.github.io/AwesomeBooksES6/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,122 @@
...
+        <nav class="navbar navbar-default">
+            <div class="container">
+              <div class="navbar-header">
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                  <span class="sr-only">Toggle navigation</span>
+                  <span class="icon-bar"></span>
+                  <span class="icon-bar"></span>
+                  <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="#">Awesome Books</a>
```
Comment (BenMukebo) [inline index.html:25]: - [x] Great jo so far 👏🏻 However, it's one of the main project requirements you Display the current date below the navigation bar. Kindly address this 👍🏻 For this requirement, you need to use the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library. See [Get the current date and time](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time). Package should be added as a .js file to the project. You have to download the library into your project  using npm then, Import it as a module: ``import { DateTime } from "luxon";``

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @ab-noori,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- The app is working, it is possible to add and remove books from the list.  ✔️   
- Correctly use `arrow` syntax for functions. ✔️ 
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @ab-noori     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R
- [x] Professional README
- [x] Nice and Neat UI design
- [x] App is working as intended









 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Comment by ab-noori** [comment]
Hi Mr. @BenMukebo and, Mr. @KayLemba,
Thanks for reviewing my project and giving precise feedback.

**Pull request merged**